### DISCUSSION TITLE
Unable to block the ip who is trying to brute force

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [wazuh/wazuh#20166] [General]
Hi, https://documentation.wazuh.com/current/user-manual/capabilities/active-response/ar-use-cases/blocking-ssh-brute-force.html i followed correctly what is mentioned there is the official documentation but even though the attacker ip is not getting blocked. i have created a custom rule also for that 

![image](https://github.com/wazuh/wazuh/assets/140578526/da2bc4d2-a0a2-4792-9032-f53039b43dd8)

and added the rule id in ossec.conf file 

![image](https://github.com/wazuh/wazuh/assets/140578526/d67f5856-226e-48c6-9dbc-045d508b2aaa)

Even though the IP address is not blocked, 
Is there any extra configuration we have to do?
So please reply! and help me to solve this!

**Comment by sellarsj** [comment]
what O/S is the machine being brute forced?

**Comment by sandraimmaculate** [comment]
Ubuntu
